cli/napi/node_api.rs

```diff
@@ -12,12 +12,13 @@ use crate::check_env;
 use deno_core::V8CrossThreadTaskSpawner;
 use deno_runtime::deno_napi::*;
 use napi_sym::napi_sym;
-use std::ptr::NonNull;
 use std::sync::atomic::AtomicBool;
 use std::sync::atomic::AtomicU8;
 use std::sync::atomic::AtomicUsize;
 use std::sync::atomic::Ordering;
 use std::sync::Arc;
+use std::sync::Condvar;
+use std::sync::Mutex;
```

Comment:
If you don't need poisoning I'd recommend using the Mutex and Condvar from parking_lot instead.